nipype/interfaces/base/support.py

```diff
@@ -9,185 +9,166 @@
 """
 from __future__ import (print_function, division, unicode_literals,
                         absolute_import)
-from builtins import object, str
+from builtins import object
 
-import os
-from copy import deepcopy
+from ... import __version__
 
-from ... import logging
-from ...utils.misc import is_container
-from ...utils.filemanip import md5, to_str, hash_infile
-iflogger = logging.getLogger('nipype.interface')
 
-
-class NipypeInterfaceError(Exception):
+class NipypeInterfaceError(RuntimeError):
```

Comment:
I would prefer not to make this API change, as previously an `except RuntimeError` would not catch this, so somebody who is counting on this to make it past that exception will be surprised. I think we should move to a proper `NipypeError` hierarchy in 2.0, though.